apps/authentication/api/face.py

```diff
@@ -128,7 +128,7 @@ def get(self, request, *args, **kwargs):
         return Response({
             "is_finished": context.get('is_finished', False),
             "success": context.get('success', False),
-            "error_message": context.get("error_message", '')
+            "error_message": _(context.get("error_message", ''))
         })
 
 
```

Comment:
I've reviewed the provided snippet (code in Python) without context and could not identify discrepancies related to technical inaccuracies, bugs, or inefficiencies. Please provide more details about what you're interested to know regarding this code.